openquake/mbt/tests/tools/test_fault_modeling_utils.py

```diff
@@ -0,0 +1,260 @@
+import json
+import unittest
+import os
+
+from openquake.mbt.tools.fault_modeler.fault_modeling_utils import *
+
+
+BASE_DATA_PATH = os.path.dirname(__file__)
+data_dir_str = '../data/tools/'
```

Comment:
``data_dir_str = os.path.join('..', 'data', 'tools')``
seems to me more OS agnostic